fd/util.go

```diff
@@ -97,7 +97,10 @@ func extractMatchMode(actionJSON *simplejson.Json) (pipeline.MatchMode, error) {
 }
 
 func extractMatchInvert(actionJSON *simplejson.Json) (bool, error) {
-	invertMatchMode := actionJSON.Get("match_invert").MustBool()
+	invertMatchMode, err := actionJSON.Get("match_invert").Bool()
```

Comment:
I need look deeper at this case, probably it can fail at upper level: it checks for err != nil, while err always null

Reply:
I revert this case